api/src/pcapi/scripts/hard_delete_venues_without_offers/main.py

```diff
@@ -0,0 +1,100 @@
+"""
+Job console documentation here: https://www.notion.so/passcultureapp/Documentation-Job-Console-769beeacd5a146de9c97b6f8ee544276
+Assumed path to the script (copy-paste in github actions):
+
+https://github.com/pass-culture/pass-culture-main/blob/ogeber/ogeber/pc-35670-hard-delete-venues-without-offers/api/src/pcapi/scripts/hard_delete_venues_without_offers/main.py
+
+"""
+
+import argparse
+import csv
+import datetime
+import logging
+import os
+import typing
+
+from sqlalchemy import exc as sa_exc
+
+from pcapi.app import app
+from pcapi.core.offerers import exceptions
+from pcapi.core.offerers.api import delete_venue
+from pcapi.core.offerers.models import Venue
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+VENUE_ID_HEADER = "Venue ID"
+
+
+def _read_csv_file(filename: str) -> typing.Iterator[dict[str, str]]:
+    namespace_dir = os.path.dirname(os.path.abspath(__file__))
+    with open(f"{namespace_dir}/{filename}.csv", "r", encoding="utf-8") as csv_file:
+        csv_rows = csv.DictReader(csv_file, delimiter=";")
+        yield from csv_rows
+
+
+def _check_venue_can_be_hard_deleted(venue_id: int, invalid_venues: list[tuple[int, str]]) -> bool:
+    venue: Venue = db.session.query(Venue).filter_by(id=venue_id).one_or_none()
+    if not venue:
+        invalid_venues.append((venue_id, "Venue not found"))
+        return False
+
+    six_month_ago = datetime.datetime.utcnow() - datetime.timedelta(days=30 * 6)
+    if (
+        venue.isPermanent
+        or venue.isOpenToPublic
+        or (venue.siret is not None and venue.siret != "")
+        or venue.has_collective_offers
+        or venue.hasOffers
+        or venue.dateCreated >= six_month_ago
```

Comment:
mmm, ce n'est pas comme ça que marchent les dates.
Il y a plus de 6 mois -> la date de création est plus ancienne qu'il y a 6 mois
```suggestion
        or venue.dateCreated <= six_month_ago
```